src/feature.js

```diff
@@ -116,12 +116,12 @@ class Feature {
       }
     );
 
-    browser.trackers.onAddException.addListener(tabId => {
+    browser.trackers.onToggleException.addListener((tabId, toggleValue) => {
       if (tabId < 0) {
         return;
       }
       const tabInfo = TabRecords.getOrInsertTabInfo(tabId);
-      tabInfo.telemetryPayload.user_added_exception = true;
+      tabInfo.telemetryPayload.user_toggled_exception = toggleValue ? 1 : 2;
```

Comment:
Should we define constants for these values?

```
// Constants for the user_toggled_exception telemetry probe
const PROTECTION_ENABLED=1
const PROTECTION_DISABLED=2
```
(Am I interpreting those values correctly?)